argo/src/Argo/DefaultMain.hs

```diff
@@ -29,10 +29,12 @@ defaultMain str app =
      realMain app opts
 
 data NetworkOptions =
-  NetworkOptions (Maybe Session) (Maybe HostName) (Maybe Port)
+  NetworkOptions (Maybe Session) (Maybe HostName) (Maybe Port) (Maybe LogOption)
```

Comment:
Does it make sense to use a record heir instead of positional arguments? 4 seems like a lot of positional params to me (although they do each have cute names to their types, so I could be easily disuaded for now I suppose)